eng/Versions.props

```diff
@@ -30,17 +30,7 @@
     <AddinVersion Condition="'$(OfficialBuildId)' != ''">$(AddinVersion).$(OfficialBuildId)</AddinVersion>
     <AddinVersion Condition="'$(OfficialBuildId)' == ''">$(AddinVersion).42424242.42</AddinVersion>
   </PropertyGroup>
-  <PropertyGroup>
-    <RestoreSources Condition="'$(DotNetBuildOffline)' != 'true'">
```

Comment:
Just want to make sure this is a reasonable change. We made a fix in 3.1 to avoid restoring these projects during test runs by importing the Versions.props. But this property prevents the root NuGet.config from being used.  Not super sure why regular projects aren't affected by this

Reply:
This is goodness. We shouldn't be using `$(RestoreSources)` anywhere. `$(RestoreAdditionalProjectSources)` helps for a few corner cases but we're not using that broadly either. NuGet.config forever.